cmd/notary/tuf.go

```diff
@@ -83,6 +83,12 @@ var cmdTUFVerifyTemplate = usageTemplate{
 	Long:  "Verifies if the data passed in STDIN is included in the remote trusted collection identified by the Global Unique Name.",
 }
 
+var cmdWitnessTemplate = usageTemplate{
+	Use:   "witness [ GUN ] <role> ...",
+	Short: "Marks roles to be re-signed the next time they're published",
```

Comment:
Is this the default mode of `witness`?

I was hoping this command could be used for addition of signatures and that that would be the primary use case for this command.